CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ stt = DeepgramSTTService(..., live_options=LiveOptions(model="nova-2-general"))
 - Fixed a `STTMuteFilter` issue that would not mute user audio frames causing
   transcriptions to be generated by the STT service.
 
+- Fixed an issue that `start_callback` was not invoked for some LLM services.
```

Comment:
super mega tiny nit, but just fyi: We usually add the new things at the top of each section.

Reply:
Great. I will fix that.